### DISCUSSION TITLE
Date problem when creating appointments

### DISCUSSION THREAD

**Discussion by Matthias1097** [liedekef/events-made-easy#432] [Q&A] [answered]
Hello,

I've been having a problem creating appointments for a few days now.
I can no longer edit the date field of the appointment.
It is greyed out, so I can't change anything there.

Does anyone know why this is and how to fix it?

Thank you very much!
![Unbenannt](https://github.com/liedekef/events-made-easy/assets/158514545/08a1c11b-de7f-46b4-be91-bca685e5b18e)

**Comment by liedekef** [comment]
Like I already answered via mail:
Check your browser console for javascript errors. Some javascript error is most probably preventing the calendar from functioning.

**Reply by Matthias1097** [reply]
Hello,
I could not find any Javascript errors. :/
Best regards

**Reply by liedekef** [reply]
The complete calendar is javascript based. If the calendar is not showing, it is due to some javascript error. If you don't know how to check yourself, feel free to send me an admin account so I can check.

**Reply by Matthias1097** [reply]
Hello Franky,
Thank you very much for your help.
I have just sent you another email about this.
Many thanks and best wishes!

**Reply by liedekef** [reply]
You seem to have a double install of EME (maybe a backup folder) ==> don't create backup folders in the WP plugin folder, it confuses wordpress. It also seems the wp-plugin folder can't be written by WP itself ...
I installed "Health Check & Troubleshooting" for testing, but couldn't go into Troubleshoot Mode because of your WP issues.
So you need to get these things fixed first

**Reply by Matthias1097** [reply]
The duplicate folder has now been deleted, but I'll have to check tomorrow why Wordpress doesn't have access to the folder.

Thanks for the help anyway!
I'll let you know as soon as I've fixed the problem.

Thank you very much and have a nice evening!

**Reply by Matthias1097** [reply]
Hello Franky,
I have now fixed the server problems and the troubleshoot mode is now working.
If you want you can have another look to see if you can find the problem.
Thank you very much!

**Reply by liedekef** [reply]
When in troubleshoot mode and only EME active, all works fine. So some plugin is interfering (or maybe providing also a javascript class called "fcalendar", causing mine not to work.
Since you have a huge amount of plugins installed, I'm not going to check them all :-)
My advise: go into troubleshoot mode, activate EME and then activate one plugin at a time and each time test if the EME calendar still works. Once you've found the conflicting plugin, you can then decide what to do ...

**Reply by Matthias1097** [reply]
Okay great, thanks for the advice.
Since this error only occurred after the Wordpress update, I had assumed that this was related to it.

I will give this a try then.

Thanks for the help and have a nice weekend!!! :)

**Reply by Matthias1097** [reply, marked as answer]
For your information:

I have found the problematic plugin:
https://de.wordpress.org/plugins/wp-whatsapp-chat/
As soon as this plugin is activated, the date fields in your plugin are greyed out.

Best regards

**Reply by liedekef** [reply]
ok, the issue is their CSS:
- first: they load it on every admin page (not good)
- second: they set "pointer-events: none" for all read-only inputs (and since the datetime selectors are readonly, this causes the actual problem). ==> they should realize this causes issues with lots of datepickers ...

For now, I forced some styles:
https://github.com/liedekef/events-made-easy/commit/5f0947da03abe3cd2c1fea4fb82c1afafcc4ae2c
